api/src/test/java/uk/gov/dhsc/htbhf/dwp/service/HouseholdVerifierUCHouseholdTest.java

```diff
@@ -0,0 +1,147 @@
+package uk.gov.dhsc.htbhf.dwp.service;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.dwp.entity.uc.UCAdult;
+import uk.gov.dhsc.htbhf.dwp.entity.uc.UCHousehold;
+import uk.gov.dhsc.htbhf.dwp.model.AddressDTO;
+import uk.gov.dhsc.htbhf.dwp.model.PersonDTO;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.dhsc.htbhf.dwp.entity.UCHouseholdFactory.aHouseholdWithNoAdultsOrChildren;
+
+public class HouseholdVerifierUCHouseholdTest {
```

Comment:
Same comments for this test around using test data factories and AssertJ.

Reply:
Done.